sat/engine2/api/secrets.go

```diff
@@ -0,0 +1,41 @@
+package api
+
+import (
+	"github.com/pkg/errors"
+	"github.com/suborbital/e2core/sat/engine2/runtime/instance"
+)
+
+func (d *defaultAPI) GetSecretValueHandler() HostFn {
```

Comment:
This should be implemented via WASI environment variables instead of a custom host function.

Reply:
Done